docs/source/index.rst

```diff
@@ -60,6 +60,7 @@ Navigate below and get started with Cloud Custodian!
    azure/contribute
    azure/authentication
    azure/usecases/index
+   azure/samplepolicies/index
```

Comment:
Looks like the hierarchy we are trying to replicate is called `/policy/resources`

Reply:
![image](https://user-images.githubusercontent.com/4087345/39827389-a373bbd8-536c-11e8-9179-85a043b50649.png)